app/src/pages/inside/projectSettingsPageContainer/projectSettingsPageContainer.scss

```diff
@@ -24,4 +24,6 @@ $Z-INDEX-PAGE-CONTAINER-HEADER: 2;
 
 .content {
   padding: 33px 32px 60px;
+  max-width: 1200px;
```

Comment:
This is a great solution, but it breaks styles in the Demo Data tab, Analyzer tab, General tab. Take a closer look at these tabs and compare the styles to the design system.
Also, note that there is an extra margin in the integrations. Because of them, integrations are not centered.
![2022-12-26_14h57_47](https://user-images.githubusercontent.com/84175555/209540203-fdf7a0d2-d261-4ec2-a2c3-89d392ce4d00.jpg)